src/amazonqGumby/entrypoint.ts

```diff
@@ -17,6 +18,7 @@ export async function processTransformByQ() {
     }
     if (transformByQState.isNotStarted()) {
         await sleep(1000) // sleep so that chat can respond first, then show input prompt
+        telemetry.codeTransform_jobIsStartedFromChatPrompt.emit()
```

Comment:
IMO having metric name this big/specific is confusing. maybe have `codeTransform_JobStatus` 
with fields being `status = [Started, In process, Ended, Closed, Cancelled, Failed]`, 
`jobId`: 
`SessionId` : 
`Result`: 
`reason`: 
`Latency`: 
`JobType`
`InvokeCause`: 
etc..

Reply:
We would like to review all of our metrics holistically as I agree that there are better ways to represent this data - this PR is just to get it consistent with metrics currently emitted by JB, as it's a business requirement that they are the same